features/04_aruba_api/command/run_command.feature

```diff
@@ -41,19 +41,24 @@ Feature: Run command
     When I run `rspec`
     Then the specs should all pass
 
-  Scenario: Relative path to executable
-    Given an executable named "bin/aruba-test-cli" with:
-    """bash
-    #!/bin/bash
-    exit 0
-    """
-    And a file named "spec/run_spec.rb" with:
+  Scenario: Executable in a relative path in the Aruba working directory
+    Given a file named "spec/run_spec.rb" with:
     """ruby
     require 'spec_helper'
 
     RSpec.describe 'Run command', type: :aruba do
-      before(:each) { run_command('bin/aruba-test-cli') }
-      it { expect(last_command_started).to be_successfully_executed }
+      before do
+        write_file 'bin/aruba-test-cli', <<~BASH
+          #!/bin/bash
+          exit 0
+        BASH
+        chmod 0x755, 'bin/aruba-test-cli'
```

Comment:
This says 0x755 where the other one says 0o755, is there a difference by choice? Or by accident?

Reply:
Seems to be a RuboCop correction. I'll equalize these.

Reply:
Nope, it was just wrong :slightly_smiling_face:. It should be `0o755` or `0755`.